projects/plugins/migration/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"private": true,
-	"description": "A Jetpack plugin that helps users to migrate their sites to WordPress.com.",
+	"description": "A WordPress plugin that helps users to migrate their sites to WordPress.com.",
 	"homepage": "https://jetpack.com",
```

Comment:
https://wordpress.com/ ..?

Reply:
Thanks for catching this, updated as suggestion!